pkgs/top-level/all-packages.nix

```diff
@@ -2466,6 +2466,8 @@ in
     libmaxminddb = null;
   };
 
+  xmlsort = perlPackages.XMLFilterSort;
```

Comment:
maybe to `aliases.nix` ?

Reply:
I thought aliases was meant for when stuff was deprecated - not for introducing a package into the top level.